pkg/api/customization/cluster/formatter.go

```diff
@@ -47,23 +47,36 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
-		canUpdateCluster := func(request *types.APIContext, clusterID string) bool {
+		canUpdateClusterFn := func(request *types.APIContext, clusterID string) bool {
 			cluster := map[string]interface{}{
 				"id": clusterID,
 			}
-			return request.AccessControl.CanDo(v3.ClusterGroupVersionKind.Group, v3.ClusterResource.Name, "update", request, cluster, resource.Schema) == nil
+			return request.AccessControl.CanDo(
+				v3.ClusterGroupVersionKind.Group,
+				v3.ClusterResource.Name,
+				"update",
+				request,
+				cluster,
+				resource.Schema) == nil
 		}
-		canCreateClusterScanObject := func() bool {
+		canCreateClusterScanObjectFn := func(request *types.APIContext, clusterID string) bool {
```

Comment:
why do we need this? isn't just `canUpdateClusterFn` sufficient?

Reply:
I think yes, isnt just checking for "update cluster" capability enough since only cluster-owner can edit a cluster? now that we don't have a separate "manage clusterscan" role.

Reply:
yup, left over from previous PR.